docs/manual/dynamicform.md

```diff
@@ -58,75 +54,83 @@ Also you can separate option's display name and value, using _${formName=default
 
 <img src="/assets/themes/zeppelin/img/screenshots/form_select_displayname.png" />
 
-<br />
 ### Creates Programmatically
 
 Some language backend uses programmatic way to create form. For example [ZeppelinContext](../interpreter/spark.html#zeppelincontext) provides form creation API
 
 Here're some examples.
 
-Text input form
-
-You can do this in Scala
+####Text input form
+<div class="codetabs">
+    <div data-lang="scala" markdown="1">
 
-```scala
+{% highlight scala %}
 %spark
 println("Hello "+z.input("name"))
-```
+{% endhighlight %}
 
-Or Python
+    </div>
+    <div data-lang="python" markdown="1">
 
-```python
+{% highlight python %}
 %pyspark
 print("Hello "+z.input("name"))
-```
+{% endhighlight %}
 
+    </div>
+</div>
 <img src="/assets/themes/zeppelin/img/screenshots/form_input_prog.png" />
 
-Text input form with default value
-
-Scala
+####Text input form with default value
+<div class="codetabs">
+    <div data-lang="scala" markdown="1">
 
-```scala
+{% highlight scala %}
 %spark
-println("Hello "+z.input("name", "sun"))
-```
+println("Hello "+z.input("name", "sun")) 
+{% endhighlight %}
 
-Python
+    </div>
+    <div data-lang="python" markdown="1">
 
-```python
+{% highlight python %}
 %pyspark
 print("Hello "+z.input("name", "sun"))
-```
+{% endhighlight %}
 
+    </div>
+</div>
 <img src="/assets/themes/zeppelin/img/screenshots/form_input_default_prog.png" />
 
-Select form
+####Select form
+<div class="codetabs">
+    <div data-lang="scala" markdown="1">
 
-Scala
-
-```scala
+{% highlight scala %}
 %spark
 println("Hello "+z.select("day", Seq(("1","mon"),
+                                        ("2","tue"),
```

Comment:
I think there's a slight indentation offset here and before, as you can see in the final html output.
Could you check?